egs2/clarity/enh_2021/local/data.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+help_message=$(cat << EOF
+Usage: $0 --clarity_root <path> [--sample_rate <sample_rate>]
+
+  required argument:
+    --clarity_root: path to clarity dataset root folder. i.e. folder which contains train and dev subfolders.
+    NOTE:
+
+
+  optional argument:
+    [--sample_rate]: 16000 (default) or 44100
+EOF
+)
+
+clarity_root=
+sample_rate=16000
+
+. utils/parse_options.sh
+
+
```

Comment:
Maybe add a check for the existence of `sox` (as it is assumed to be available in `local/prep_data.py`) like the following?
```bash
! command -v sox &>/dev/null && echo "sox: command not found" && exit 1;
```